lib/util/devops.js

```diff
@@ -2,7 +2,7 @@ const File = require('./file');
 const path = require('path');
 const inquirer = require('inquirer');
 const Util = require('./util');
-const git = require('simple-git/promise')();
+const git = require('simple-git')();
```

Comment:
why change how we use simple-git?

Reply:
as before ;) by default supports promises now